pro/src/components/LegalInfos/LegalInfos.tsx

```diff
@@ -3,7 +3,8 @@ import { useLocation } from 'react-router-dom'
 
 import { Events } from 'core/FirebaseEvents/constants'
 import useAnalytics from 'hooks/useAnalytics'
-import Icon from 'ui-kit/Icon/Icon'
+import { ReactComponent as IcoExternalSiteRedFilled } from 'icons/ico-external-site-red-filled.svg'
+import { ReactComponent as IcoMail } from 'icons/ico-mail.svg'
```

Comment:
proposition: ajout dans `icons/index.ts` 
```js
export { ReactComponent as IcoExternalSiteRedFilled } from 'icons/ico-external-site-red-filled.svg'
export { ReactComponent as IcoMail } from 'icons/ico-mail.svg'
```
puis 
```suggestion
import { IcoExternalSiteRedFilled, IcoMail } from 'icons'
```

Reply:
merci c'est plus propre donc je l'ai ajouté